### ISSUE TITLE
Improve descriptions if the title of an axis is disabled

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#6454]
Take for example

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "data": {
    "values": [
      {"a": "A", "b": 28}, {"a": "B", "b": 55}, {"a": "C", "b": 43},
      {"a": "D", "b": 91}, {"a": "E", "b": 81}, {"a": "F", "b": 53},
      {"a": "G", "b": 19}, {"a": "H", "b": 87}, {"a": "I", "b": 52}
    ]
  },
  "mark": "bar",
  "encoding": {
    "x": {"field": "a", "type": "ordinal", "axis": {"title": null}},
    "y": {"field": "b", "type": "quantitative", "axis": {"title": null}}
  }
}
```

Since the title is disabled, the descriptions of the bars should be just `A 28` instead of `a: A; b: 28`.

To implement this, look at `description` in `aria.ts`.

**Comment by kanitw** [comment]
I don't like this suggestion. 

You may want to hide the axis title simply because the chart title already provides redundant information.  

It's probably better to be able to configure the description directly.

**Comment by domoritz** [comment]
Okay, let's close this then.
